### DISCUSSION TITLE
Winlogbeat and Logstash

### DISCUSSION THREAD

**Discussion by jjcn05** [Security-Onion-Solutions/securityonion#3336] [Unsupported Versions] [answered]
Hi, 
Hoping someone will be able to assist me with this issue. Been trying to troubleshoot for the past two days...

Recently started playing around with SO to learn more about logs and threat hunting for my new SOC job. So I decided to try and implement sysmon with winlogbeats...

### Background on my Home Lab

- Mini pc with 32 gb , 1 NIC , running ESXI.
- running pfsense , SO, windows client , ubuntu client
- all traffic will go through the pfsense (WAN --> LAN ) 
- [WAN / Management](https://imgur.com/3GLO3S9)
- [LAN](https://imgur.com/62A5RyI)

### SO Setup

- installed standalone, due to logstash requirement 
- Port mirroring is working properly, seeing network logs 
- But don't think it's setup properly since my velociraptor is not seeing clients ( integrated to SO ) 
- followed the documentation [here](https://docs.securityonion.net/en/2.3/beats.html) 
- dashboard disabled : [here](https://imgur.com/BCYiLlT) 
- output.logstash enabled and re-directing to correct host : [here](https://imgur.com/e0pk5XW) 
  - I can ping it from client
  - But I can't telnet into it... ( Did so-allow )
- output.elasticsearch disabled : [here](https://imgur.com/v5yAyAe) 
- did so-status everything is enabled and running properly...

Not sure why I can't telnet into 10.0.0.200:5504 ( logstash ) but I did do so-allow for logstash/elastic/analyst just to be safe...
I thought it was my client AV... so I disabled Windows Defender...
Tried to see if it was my pfsense FW rule... so I allowed every VLAN to talk to 10.0.0.200 (any port ) 

Error code that I'm getting [here](https://imgur.com/w5oXxuD) 
- "Attempting to reconnect to backoff ..."

I'm out of ideas... lol someone help

**Comment by weslambert** [comment, marked as answer]
Are you making sure to send it to 5044, and not 5504?

**Reply by jjcn05** [reply]
Thanks for replying back.. I think it's fixed? I'm getting winlogbeat logs now. Not sure what happened ... I kinda gave up and took a look today and its working.
